scenarios/command/python/vehicle_03.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+import sys
+from connectionsettings import connection_settings as cs
+from argparse import ArgumentParser
+import logging
+import datetime
+import time
+import uuid
+import ssl
+import threading
+import paho.mqtt.client as mqtt
+from paho.mqtt.properties import Properties
+from paho.mqtt.packettypes import PacketTypes
+from concurrent.futures import ThreadPoolExecutor
+
+REQUEST_TOPIC_PATTERN = "vehicles/{targetClientId}/command/{commandName}/request"
+RESPONSE_TOPIC_PATTERN = "vehicles/{targetClientId}/command/{commandName}/response"
+
+parser = ArgumentParser()
+parser.add_argument("--env-file", help="path to the .env file to use")
+args = parser.parse_args()
+
+tpe = ThreadPoolExecutor()
+
+logging.basicConfig(level=logging.DEBUG)
+
+connected_cond = threading.Condition()
+connected_prop = False
+connection_error = None
+subscribed_cond = threading.Condition()
+subscribed_prop = False
+
+def on_connect(client, _userdata, _flags, rc, _properties):
+    global connected_prop
+    global connection_error
+    print("Connected to MQTT broker")
+    # # In Paho CB thread.
+    with connected_cond:
+        if rc == mqtt.MQTT_ERR_SUCCESS:
+            connected_prop = True
+        else:
+            connection_error = Exception(mqtt.connack_string(rc))
+        connected_cond.notify_all()
+
+def on_subscribe(client, _userdata, mid, _reason_codes, _properties):
+    global subscribed_prop
+    print(f"Subscribe for message id {mid} acknowledged by MQTT broker")
+    # # In Paho CB thread.
+    with subscribed_cond:
+        subscribed_prop = True
+        subscribed_cond.notify_all()
+
+def on_disconnect(_client, _userdata, rc, _properties):
+    print("Received disconnect with error='{}'".format(mqtt.error_string(rc)))
+    global connected_prop
+    # # In Paho CB thread.
+    with connected_cond:
+        connected_prop = False
+        connected_cond.notify_all()
+
+def send_unlock_response(mqtt_client, correlation_id):
+    topic = RESPONSE_TOPIC_PATTERN.format(targetClientId="vehicle03", commandName="unlock")
+    payload = "placeholder"
+    msg_prop = Properties(PacketTypes.PUBLISH)
+    msg_prop.UserProperty = ("Succeed", "True")
```

Comment:
Should be in payload, not user property

Reply:
we are using properties to indicate status.. what needs to be in the payload?

Reply:
In the C# sample, Succeed: True is in the payload, and Status: 200 is in a user property - it seems reasonable to me if we don't need to have anything in the payload on the response though